doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -41,15 +41,22 @@ Synopsis
 Options
 ~~~~~~~
 
-  -f, --format <html|xml|xml:html>
-                           Specifies whether to generate HTML or XML output.
-                           `xml:html` format means generating an XML output
-                           where keyword documentation is converted to HTML
-                           regardless of the original documentation format. The
-                           default output format is got from the output file
-                           extension so that :file:`*.html` -> `html`,
-                           :file:`*.xml` -> `xml` and :file:`*.libspec`->
-                           `xml:html`.
+  -f, --format <html|xml> Specifies whether to generate an HTML or XML
+                          output file. The default output `format` and the 
+                          default output `specdocformat` is got from the
+                          output file extension.
+                          :file:`*.html`    -> `-f HTML -s HTML`
+                          :file:`*.xml`     -> `-f XML  -s RAW`
+                          :file:`*.libspec` -> `-f XML  -s HTML`
```

Comment:
I don't think there's a need to talk about spec doc format in this help text at all.